modules/react/button/stories/icon-button/IconButton.stories.mdx

```diff
@@ -79,6 +80,19 @@ backgrounds.
 
 <ExampleCodeBlock code={InverseFilled} />
 
+### Mirroring Icons
+
+Sometimes icons should be mirrored for bidirectional support. Setting the `shouldMirrorIcon` prop to
+`true` will transform the icon's x-axis to mirror the graphic. This is often useful when there is no
+equivalent mirrored icon available. Below is an example of the `unorderedListIcon` being rendered in
+its default state with an icon showing a left-to-right bulleted list. Beside it is the same icon
+that has been mirrored to show a right-to-left bulleted list.
+
+Note that while the `aria-label` text in this example did not need to be updated, if the label
+contains directional descriptors such as "right" and "left," the text should also be updated.
+
+<ExampleCodeBlock code={MirroredIcon} />
+
```

Comment:
This wasn't in the original PR, but I thought we should add more context on why and when mirroring icons is valuable.